src/services/sync/strategies/LDAPSystemSyncer.js

```diff
@@ -42,7 +42,7 @@ class LDAPSystemSyncer extends Syncer {
 	}
 
 	getSystems() {
-		return this.app.service('systems').find({ query: { type: 'ldap' }, paginate: false })
+		return this.app.service('systems').find({ query: { type: 'ldap', active: true }, paginate: false })
```

Comment:
`active` is an attribute within the `ldapOptions`, not the root scope of a system, so the query should be `{ type: 'ldap', 'ldapConfig.active': true }`

Reply:
sounds good. Can you update?